apps/organization/mutation.py

```diff
@@ -0,0 +1,31 @@
+import graphene
+from organization.schema import OrganizationType
+from organization.serializers import OrganizationGqSerializer
+
+from utils.graphene.mutation import (
+    generate_input_type_for_serializer,
+    GrapheneMutation
+)
+
+from .models import Organization
+
+OrganizationInputType = generate_input_type_for_serializer(
+    'OrganizationInputType',
+    serializer_class=OrganizationGqSerializer,
+)
+
+
+class OrganizationCreate(GrapheneMutation):
+    class Arguments:
+        data = OrganizationInputType(required=True)
+    model = Organization
+    result = graphene.Field(OrganizationType)
+    serializer_class = OrganizationGqSerializer
+
+    @classmethod
+    def check_permissions(cls, info, **kwargs):
+        return True
```

Comment:
Can we verify this permission? @sudan45

Reply:
In deep, all users have ability to create Organizations if needed.
Each organization have `verified` field, which here is set to False, From admin panel, maintainers can verify/merge organizations as required.